### PR TITLE
 Add ability to configure CI management for POM

### PR DESCRIPTION
**Pull request by koral--**

### CODE REVIEW

File: subprojects/shipkit/src/test/groovy/org/shipkit/internal/gradle/util/PomCustomizerTest.groovy [hunk 1]
```diff
@@ -186,6 +186,41 @@ class PomCustomizerTest extends Specification {
 """
     }
 
+    def "CI management from configuration"() {
```
Comment (mstachniuk) [inline subprojects/shipkit/src/test/groovy/org/shipkit/internal/gradle/util/PomCustomizerTest.groovy]: please also cover a default value ciManagement url and system

Comment (koral--) [reply subprojects/shipkit/src/test/groovy/org/shipkit/internal/gradle/util/PomCustomizerTest.groovy]: Added in next commit

**Comment by koral--** [comment]
Merging this since all the review comments has been fixed.
Skipping release due to size of change.

**Comment by mockitoguy** [comment]
Nice change!

**Pull request merged**